docs/spec_checklist.md

```diff
@@ -0,0 +1,28 @@
+# StableHLO Specification Checklist
+
+In this document, we summarize the guidelines for reviewing changes to the
+specification. At the moment, these changes typically involve checking multiple
+things in multiple sources, so this document summarizes them all to simplify
+reviews:
+
+  1. Check if the section title matches the mnemonic in
+     [the ODS](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.td).
+  1. Check if the "Semantics" section matches XLA's
+     [Operation Semantics](https://www.tensorflow.org/xla/operation_semantics).
+  1. Check if the "Operands" section lists items in the same order as the ODS.
+  1. Check whether the "Constraints" section:
+      1. Matches XLA's
+         [shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/shape_inference.cc).
+      1. Matches XLA's
+         [hlo_verifier.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/hlo_verifier.cc).
+      1. Matches the ODS.
+      1. Matches
+         [StablehloOps.cpp](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.cpp).
+      1. If there are any mismatches, check that there are corresponding tickets.
+      1. If the corresponding parts of the ODS and StablehloOps.cpp match the
```

Comment:
In the description section of ODS add a link to the section in spec_draft.md which specifies the op.

Reply:
Good catch! I was very confident that I mentioned that already, but it looks like I haven't :)